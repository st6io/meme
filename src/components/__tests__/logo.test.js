import React from 'react';
import { mount } from 'enzyme';

import Logo from '../logo';

describe('<Logo />', () => {
  it('should match snapshot', () => {
    const component = mount(<Logo />);
    expect(component).toMatchSnapshot();
  });
});
