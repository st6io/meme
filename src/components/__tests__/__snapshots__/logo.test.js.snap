// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should match snapshot 1`] = `
<Logo>
  <ForwardRef
    width={70}
  >
    <svg
      width={70}
    >
      logo.svg
    </svg>
  </ForwardRef>
</Logo>
`;
